docs-source/content/userguide/managing-domains/domain-lifecycle/scaling.md

```diff
@@ -87,8 +87,27 @@ For example, when using `curl`:
 curl -v -k -H X-Requested-By:MyClient -H Content-Type:application/json -H Accept:application/json -H "Authorization:Bearer ..." -d '{ "managedServerCount": 3 }' https://.../scaling
 ```
 
-If you omit the header, you'll get a `400 (bad request)` response. If you omit the Bearer Authentication header, then you'll get a `401 (Unauthorized)` response.
+If you omit the header, you'll get a `400 (bad request)` response. If you omit the Bearer Authentication header, then you'll get a `401 (Unauthorized)` response.  If the service account or user associated to the `Bearer` token does not have permission to `patch` the WebLogic domain resource, you'll get a `403 (Forbidden)` response.
```

Comment:
associated to -> associated with
resource, you'll -> resource, then you'll

Reply:
Fixed all review comments on documentation.